modules/io/adaptors/parse_bytes_to_dataframe.cc

```diff
@@ -130,7 +212,9 @@ int main(int argc, const char** argv) {
     if (status.ok()) {
       LOG(INFO) << "consumer: buffer size = " << buffer->size();
       std::shared_ptr<arrow::Table> table;
-      ParseTable(&table, buffer, delimiter[0], header_row, col_names);
+      VINEYARD_CHECK_OK(ParseTable(&table, buffer, delimiter[0], header_row,
+                                   columns, column_types, original_columns,
+                                   include_all_columns));
```

Comment:
Here it shouldn't be `VINEYARD_CHECK_OK`, since if it fails, we need to `ReportStatus("error", ...)`.